### PR TITLE
Use display module when generating CTE

### LINKED ISSUE #649: use display module when generrating CTE
when generating a CTE, we mention the user which snippets we're using:

image

`Generating CTE with ...`

we should change the message so it uses the display.py module

### PR DESCRIPTION
**Pull request by tl3119**
## Describe your changes
Remove the print message.
Use display module to show the message instead when generating CTE

## Issue number

Closes #649 

## Checklist before requesting a review

- [x] Performed a self-review of my code
- [x] Formatted my code with [`pkgmt format`](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#linting-formatting)
- [x] Added [tests](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#testing) (when necessary).
- [ ] Added [docstring](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#documenting-changes-and-new-features) documentation and update the [changelog](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#changelog) (when needed)

----
:books: Documentation preview :books:: https://jupysql--683.org.readthedocs.build/en/683/

### CODE REVIEW

**Comment by yafimvo** [comment]
@tl3119 Is it ready for review? If so, please make sure it passes the CI.

**Comment by tl3119** [comment]
> @tl3119 Is it ready for review? If so, please make sure it passes the CI.

It is ready for review now.

File: src/sql/magic.py [hunk 1]
```diff
@@ -360,7 +360,7 @@ def interactive_execute_wrapper(**kwargs):
         with_ = self._store.infer_dependencies(command.sql_original, args.save)
         if with_:
             command.set_sql_with(with_)
-            print(f"Generating CTE with stored snippets : {', '.join(with_)}")
+            display.message("Generating CTE using display.py module")
```
Comment (edublancas) [inline src/sql/magic.py:363]: we should use the display.py module but the message should stay the same

**Review by edublancas — CHANGES_REQUESTED** [review]

File: src/sql/magic.py (hunk 1 above)
Comment (tl3119) [reply src/sql/magic.py]: Got it. Update it now.

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,7 @@
 * [Fix] Fixed CI issue by updating `invalid_connection_string_duckdb` in `test_magic.py` (#631)
 * [Fix] Refactored `ResultSet` to lazy loading (#470)
 * [Fix] Removed `WITH` when a snippet does not have a dependency (#657)
+* [Fix] Used display module when generrating CTE (#649)
```
Comment (tonykploomber) [inline CHANGELOG.md:12]:
```suggestion
* [Fix] Used display module when generating CTE (#649)
```

**Review by tonykploomber — CHANGES_REQUESTED** [review]
typo

**Comment by tonykploomber** [comment]
@tl3119 
1. Let's add some description into _Describe your changes_, proper description is required to be reviewed
2. You can click "Resolve Conversation" if you feel the comment is addressed properly

**Comment by tl3119** [comment]
> @tl3119
> 
> 1. Let's add some description into _Describe your changes_, proper description is required to be reviewed
> 2. You can click "Resolve Conversation" if you feel the comment is addressed properly

Sure!

File: src/sql/magic.py [hunk 3]
```diff
@@ -360,7 +360,7 @@ def interactive_execute_wrapper(**kwargs):
         with_ = self._store.infer_dependencies(command.sql_original, args.save)
         if with_:
             command.set_sql_with(with_)
-            print(f"Generating CTE with stored snippets : {', '.join(with_)}")
+            display.message(f"Generating CTE with stored snippets : {', '.join(with_)}")
```
Comment (edublancas) [inline src/sql/magic.py:363]:
```suggestion
            display.message(f"Generating CTE with stored snippets: {', '.join(with_)}")
```

**Review by edublancas — CHANGES_REQUESTED** [review]

File: src/sql/magic.py (hunk 3 above)
Comment (tl3119) [reply src/sql/magic.py]: Update it

**Review by yafimvo — APPROVED** [review]
lgtm. 

@tl3119 Next time please remember to include a link to the documentation preview in the PR description

**Comment by tl3119** [comment]
> lgtm.
> 
> @tl3119 Next time please remember to include a link to the documentation preview in the PR description

Sure!

**Review by edublancas — APPROVED** [review]

**Pull request merged**